install/ontology/general.rdf

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<rdf:RDF
+        xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+        xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+        xmlns:lom="http://www.tao.lu/Ontologies/TAOLom.rdf#"
+        xml:base="http://www.tao.lu/Ontologies/TAOLom.rdf#"
+        xmlns:generis="http://www.tao.lu/Ontologies/generis.rdf#"
+        xmlns:widget="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#"
+>
+
+    <!--LOM properties for General category-->
+
+    <!--General identifier-->
+    <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAOLom.rdf#general-identifier">
+        <rdf:type rdf:resource="http://www.tao.lu/Ontologies/TAOLom.rdf#lomProperty"/>
+        <rdfs:label xml:lang="en-US"><![CDATA[LOM General identifier]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[LOM identifier into General category]]></rdfs:comment>
+        <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#AssessmentContentObject"/>
+        <rdfs:range rdf:resource="http://www.w3.org/2000/01/rdf-schema#literal"/>
+        <lom:lomCategoryProperty rdf:resource="http://www.tao.lu/Ontologies/TAOLom.rdf#General"/>
+        <generis:validationRule><![CDATA[notEmpty]]></generis:validationRule>
+        <widget:widget rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#Readonly"/>
```

Comment:
I understand how you went to such a choice, however, I would allow end-users to change the identifier, even if it will not happen a lot in real-life.